services/oada_upload/src/oada_upload.ts

```diff
@@ -15,15 +15,19 @@ declare global {
 // Packages
 import { assert } from '@sindresorhus/is';
 import { config } from 'dotenv';
-import { Client } from 'pg';
+import pglib from 'pg';
 import moment from 'moment';
 import ksuid from 'ksuid';
 import { randomBytes } from 'crypto'
 import pMap from 'p-map';
 import { connect } from '@oada/client';
 import { V1 as Location } from '@oada/types/oada/isoblue/location/v1';
+import sleep from 'atomic-sleep';
+import GeohashLib from 'latlon-geohash';
 
-import { isoblueDataTree } from './trees';
+import { isoblueDataTree } from './trees.js';
```

Comment:
.js ?

Reply:
ts-node refused to import it if you left off the .js.  I'm not sure if that's also still required for building with tsc instead of ts-node, but it was related to the mixture of commonjs and esm between the imported libraries and the code.  Maybe this can be removed now.